backend/btrixcloud/models.py

```diff
@@ -128,8 +128,8 @@ class CrawlConfigIn(BaseModel):
     autoAddCollections: Optional[List[UUID4]] = []
     tags: Optional[List[str]] = []
 
-    crawlTimeout: Optional[int] = 0
-    maxCrawlSize: Optional[int] = 0
+    crawlTimeout: int = 0
+    maxCrawlSize: int = 0
```

Comment:
Hm, this would require that every new config must include `crawlTimeout` and `maxCrawlSize`, which I think will be a breaking change for folks. Maybe we could keep these optional in the input but make sure they're set to a non-None value in the endpoint?

Reply:
Wait, nope, was confused about `Optional`! Disregard.